SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1789,8 +1786,7 @@ uint32 USpatialNetDriver::GetNextReliableRPCId(AActor* Actor, ESchemaComponentTy
 			// We previously used to receive RPCs of this type, now we're about to send one, so we reset the reliable RPC index.
 			// This should only be possible for CrossServer RPCs.
 			check(RPCType == SCHEMA_CrossServerRPC);
-			UE_LOG(LogSpatialOSNetDriver, Verbose, TEXT("Actor %s, object %s: Used to receive reliable CrossServer RPCs from worker %s, now about to send one. The entity must have crossed boundary."),
-				*Actor->GetName(), *TargetObject->GetName(), *RPCIdEntry->WorkerId);
+			UE_LOG(LogSpatialOSNetDriver, Verbose, TEXT("Actor %s, object %s: Used to receive reliable CrossServer RPCs from worker %s, now about to send one. The entity must have crossed boundary."), *Actor->GetName(), *TargetObject->GetName(), *RPCIdEntry->WorkerId);
```

Comment:
Any way to keep user formatting in places like these?

Reply:
You can surround any code with this to switch formatting off and then turn it back on.
```
// clang-format off
...
// clang-format on
```